redaxo/src/core/lib/util/string.php

```diff
@@ -224,6 +224,17 @@ public static function sanitizeHtml(string $html): string
             $antiXss->removeNeverAllowedStrAfterwards(['&lt;script&gt;', '&lt;/script&gt;']);
         }
 
+        /** @psalm-taint-escape html */
         return $antiXss->xss_clean($html);
     }
+
+    /**
+     * Escapes a given string, while leaving some basic html markup which should be safe regarding xss.
+     */
+    public static function escapeHtml(string $htmlString):string {
```

Comment:
Ich finde das etwas verwirrend:
`rex_escape($string, 'html')` vs `rex_string::escapeHtml($string)`.

Vom Namen her würde ich vermuten, dass die wohl das gleiche tun.

Ggf. statt neuer Methode, nur in `rex_escape` eine neue Strategy einführen?
Wobei die Frage ist, wie man sie nennt. `html_allow_simple_tags`, irgendwie sowas vielleicht?

Reply:
hab jetzt mal `rex_escape($message, 'html_simplified');` eingebaut